boltdesignsystem/bolt
# [DO NOT MERGE] DS 615 Convert 'string' to 'any' in schema where appropriate

## Jira

https://pegadigitalit.atlassian.net/browse/DS-615

## Summary

`String` props were changed to `any` across all `.schema`'s files.

## Details

`String` props were changed to `any` across all `.schema`'s in all:

- bolt elements
- bolt components

Props converting were omitted if an enumeration was found in a code block.

## How to test

Pull the branch, Make sure all `string` props were changed to `any`. Check if there is more to change or if some of them were changed unnecessarily.
